agent/src/agent/streamsets/manager.py

```diff
@@ -139,7 +131,8 @@ def add_empty(s: StreamSets):
         if s.id not in pipeline_streamsets:
             pipeline_streamsets[s.id] = 0
     # choose streamsets with the lowest number of pipelines
-    pipeline_streamsets = pipeline.repository.count_by_streamsets()
+    pipeline_streamsets = streamsets.repository.count_pipelines_by_streamsets()
+    logger.info(pipeline_streamsets)
```

Comment:
is this logger important? In my pr for sdc channel I removed the function count_by_streamsets()
and as I understand count_pipelines_by_streamsets() produces same result as count_by_streamsets()?

Reply:
yeah, i'll remove the logger

the result of count_pipelines_by_streamsets is the same, the query is made differently.Now streamsets might be empty (because we don't have monitoring pipelines anymore) so we need to do `SELECT streamsets  OUTER JOIN pipelines`. The old query selects from pipelines table and skips streamsets instances which don't have pipelines yet

Reply:
ok